tests/test_handler.py

```diff
@@ -13,50 +14,52 @@
     IUriHandler,
     UriHandler,
     _build_uri_handler,
-    get_uri_handler
+    get_uri_handler,
```

Comment:
Moet die komma hier staan? En deze splitsen we normaal ook op met telkens een `from urihandler.handler import...` er voor.